atom/browser/native_window_views_win.cc

```diff
@@ -299,6 +300,59 @@ bool NativeWindowViews::PreHandleMSG(UINT message,
   }
 }
 
+void NativeWindowViews::HandleSizingEvent(WPARAM w_param, LPARAM l_param) {
+  double aspect_ratio = GetAspectRatio();
+  if (fabs(aspect_ratio) < 0.0001) {
```

Comment:
On mac, we ignore negative aspect ratios and I think we should here too. I disagree with @ckerr that we should be checking `fabs` here and I think it's fine to compare for exactly zero. On mac, the code does roughly this:
```suggestion
  if (aspect_ratio <= 0.0) {
```